File: mobilebowling/app/src/main/java/com/cardflight/mobilebowling/model/Game.java
package com.cardflight.mobilebowling.model;

import java.util.ArrayList;
import java.util.List;

public class Game {

    public static final int NUMBER_OF_FRAMES = 10;
    public static List<Frame> userFrames = new ArrayList<Frame>();

    private int totalScore = 0;

    //Getters and Setters
    public Frame[] getFrames() {
        return userFrames;
    }
    public void setFrames(Frame[] frames) {
        this.userFrames = userFrames;
    }
    public int getTotalScore() {
        return this.totalScore;
    }

}

File: mobilebowling/app/src/main/java/com/cardflight/mobilebowling/model/Game.kt
package com.cardflight.mobilebowling.model

import java.util.*

class Game {
    private lateinit var userFrames: List<Frame>
    var totalScore = 0
        get() {
            for (frame in userFrames) {
                field += frame.totalFrameScore
            }
            return field
        }
        private set
    var frames: List<Frame>
        get() = userFrames
        set(userFrames) {
            this.userFrames = userFrames
        }

    companion object {
        const val NUMBER_OF_FRAMES = 10
    }
}